src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -540,7 +544,7 @@ protected virtual int ExecuteVisualStudio2000(ref Guid pguidCmdGroup, uint comma
 
         private void ExecuteEncapsulateField(ITextBuffer subjectBuffer, IContentType contentType, Action executeNextCommandTarget)
         {
-            CurrentHandlers.Execute<EncapsulateFieldCommandArgs>(contentType,
+            CurrentHandlers.Execute(contentType,
                 args: new EncapsulateFieldCommandArgs(ConvertTextView(), subjectBuffer),
```

Comment:
It's not relevant other than cleaning up the file to remove the many unnecessary (and faded) generic arguments. When adding a new handler, I found that we had been inconsistent and decided to make it so.